docs/devel/networking.md

```diff
@@ -1,4 +1,14 @@
 # VMI Networking
+Before we explain how KubeVirt performs VM networking configuration, it is
+paramount we separate the pod networking configuration and the  VM networking
+configuration concepts. Pod networking is out of the scope of this document;
+k8s is responsible to set up pod networking according to its configuration, via
```

Comment:
I think we should use the full name. k8s is degrading and may be confusing for some. No reason to save characters in documentation.

Reply:
Done